packages/evm/tests/evidence.test.ts

```
@@ -1,73 +1,16 @@
-  let wallet: ethers.Wallet;
-  let evidenceContract: ReturnType<
-    typeof getEvidencePrecompileEthersV6Contract
-  >;
+describe.skip('Evidence Precompile Tests', () => {
 
-  beforeAll(async () => {
-    const [, walletInstance] = setupProviderAndWallet();
-    wallet = walletInstance;
-    evidenceContract = getEvidencePrecompileEthersV6Contract(wallet);
-    console.log('Wallet address:', wallet.address);
+  it('should get all evidence with pagination', () => {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)